fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1477,6 +1475,15 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static long tablet_sched_consecutive_full_clone_delay_sec = 180; // 3min
 
+    /**
+     * Doing distribution balance may break the disk size balance,
+     * but the maximum gap between disks cannot exceed tablet_sched_distribution_balance_threshold_ratio * tablet_sched_balance_load_score_threshold
+     * If there are tablets in the cluster that are constantly balancing from A to B and B to A, reduce this value.
+     * If you want the tablet distribution to be more balanced, increase this value.
+     */
+    @ConfField(mutable = true)
+    public static double tablet_sched_distribution_balance_threshold_ratio = 0.5;
```

Comment:
how about `tablet_sched_num_based_balance_threshold_ratio`? balance can be number based or size based

Reply:
done